modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OrthologAnnotation_conf.pm

```diff
@@ -0,0 +1,185 @@
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
+
+=head1 DESCRIPTION
+
+=head1 AUTHOR 
+
+ mchakiachvili@ebi.ac.uk
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
+
+use strict;
+use warnings;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        # inherit other stuff from the base class
+        %{$self->SUPER::default_options},
+        'registry'         => '',
+        # 'output_base'      => '/nfs/ftp/pub/databases/ensembl/projections/',
+        'output_base'      => '/hps/nobackup2/ftp/pub/databases/ensembl/projections/',
+        'output_dirname'   => $self->o('division'),
+        'method_link_type' => 'ENSEMBL_ORTHOLOGUES',
+        'cleanup_dir'      => 1,
+        'species'          => '',
+        'antispecies'      => '',
+        'division'         => '',
+        'run_all'          => 0,
+    }
+} ## end sub default_options
+# override the default method, to force an automatic loading of the registry in all workers
+sub beekeeper_extra_cmdline_options {
+	my ($self) = @_;
+	return ' -reg_conf ' . $self->o('registry'),;
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+    return [
+        # inheriting database and hive tables' creation
+        @{$self->SUPER::pipeline_create_commands},
+        'mkdir -p ' . $self->o('output_base'),
+    ]
+} ## end sub pipeline_create_commands
+
+sub pipeline_wide_parameters {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::pipeline_wide_parameters},
+    }
+} ## end sub pipeline_wide_parameters
+
+sub pipeline_analyses {
+    my ($self) = @_;
+
+    return [
+        {
+            -logic_name      => 'SpeciesFactory',
+            -module          => 'Bio::EnsEMBL::Production::Pipeline::Common::SpeciesFactory',
+            -max_retry_count => 1,
+            -input_ids       => [ {} ],
+            -parameters      => {
+                species     => $self->o('species'),
+                antispecies => $self->o('antispecies'),
+                division    => $self->o('division'),
+                run_all     => $self->o('run_all'),
+            },
+            -rc_name         => 'default',
+            -flow_into       => [ 'SpeciesNoOrthologs' ],
+            -hive_capacity   => 1
+        },
+        #TODO RE introduce Orthologs dumping here when Compara homologies available
```

Comment:
If you're going to introduce orthologs back in here, that will make it extremely similar to the current DumpOrthologs pipeline - would be better to make that single pipeline flexible enough to do both use cases, rather than maintain two almost-identical configs.